src/main/java/frc/robot/utils/math/EqualsUtil.java

```diff
@@ -7,7 +7,6 @@ public static boolean epsilonEquals(double a, double b, double epsilon) {
     return (a - epsilon <= b) && (a + epsilon >= b);
   }
 
-  /* Really small epsilon, just a reminder */
```

Comment:
Why was this comment removed and not just improved? @AnshulBompada

Reply:
I accidentally deleted that, I am guessing. I will undo that.